src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java

```diff
@@ -141,8 +160,7 @@ public class CaseData {
     @CCD(
         access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
     )
-    @JsonIgnore
-    private List<CaseworkerCICDocument> caseDocuments;
+    private List<ListValue<CaseworkerCICDocument>> caseDocuments;
```

Comment:
Changing the type of this case field, migration required

Reply:
Will revert this change as it is not part of Case flags feature. Mistakenly added as part of merge.